app/facades/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class FoodFacade
+  def self.search_foods(search_term)
+    foods = FoodService.search_foods(search_term)
+    foods[0..9].map do |data|
+      Food.new(data)
+    end
+  end
+  def self.total_results(search_term)
+    count = FoodService.total_results(search_term)
```

Comment:
Another option so that you don't have to call the API twice: Retrieve the total hits in the `search_foods` method and return both the collection of POROs *and* the total count in an array: `[foods, totalCount]`